import unittest

class TestHandleErrorOfSwift(unittest.TestCase):
    def test_authenticate_swift(self):
        """
        OK
        :return:
        """
        pass

    def test_check_and_create_container(self):
        """
        ok
        :return:
        """
        pass